task-json/countries.js

```diff
@@ -0,0 +1,270 @@
+    {name: 'Saint Lucia', code: 'LC'}, 
+    {name: 'Saint Pierre and Miquelon', code: 'PM'}, 
+    {name: 'Saint Vincent and the Grenadines', code: 'VC'}, 
+    {name: 'Samoa', code: 'WS'}, 
+    {name: 'San Marino', code: 'SM'}, 
+    {name: 'Sao Tome and Principe', code: 'ST'}, 
+    {name: 'Saudi Arabia', code: 'SA'}, 
+    {name: 'Senegal', code: 'SN'}, 
+    {name: 'Serbia and Montenegro', code: 'CS'}, 
+    {name: 'Seychelles', code: 'SC'}, 
+    {name: 'Sierra Leone', code: 'SL'}, 
+    {name: 'Singapore', code: 'SG'}, 
+    {name: 'Slovakia', code: 'SK'}, 
+    {name: 'Slovenia', code: 'SI'}, 
+    {name: 'Solomon Islands', code: 'SB'}, 
+    {name: 'Somalia', code: 'SO'}, 
+    {name: 'South Africa', code: 'ZA'}, 
+    {name: 'South Georgia and the South Sandwich Islands', code: 'GS'}, 
+    {name: 'Spain', code: 'ES'}, 
+    {name: 'Sri Lanka', code: 'LK'}, 
+    {name: 'Sudan', code: 'SD'}, 
+    {name: 'Suriname', code: 'SR'}, 
+    {name: 'Svalbard and Jan Mayen', code: 'SJ'}, 
+    {name: 'Swaziland', code: 'SZ'}, 
+    {name: 'Sweden', code: 'SE'}, 
+    {name: 'Switzerland', code: 'CH'}, 
+    {name: 'Syrian Arab Republic', code: 'SY'}, 
+    {name: 'Taiwan, Province of China', code: 'TW'}, 
+    {name: 'Tajikistan', code: 'TJ'}, 
+    {name: 'Tanzania, United Republic of', code: 'TZ'}, 
+    {name: 'Thailand', code: 'TH'}, 
+    {name: 'Timor-Leste', code: 'TL'}, 
+    {name: 'Togo', code: 'TG'}, 
+    {name: 'Tokelau', code: 'TK'}, 
+    {name: 'Tonga', code: 'TO'}, 
+    {name: 'Trinidad and Tobago', code: 'TT'}, 
+    {name: 'Tunisia', code: 'TN'}, 
+    {name: 'Turkey', code: 'TR'}, 
+    {name: 'Turkmenistan', code: 'TM'}, 
+    {name: 'Turks and Caicos Islands', code: 'TC'}, 
+    {name: 'Tuvalu', code: 'TV'}, 
+    {name: 'Uganda', code: 'UG'}, 
+    {name: 'Ukraine', code: 'UA'}, 
+    {name: 'United Arab Emirates', code: 'AE'}, 
+    {name: 'United Kingdom', code: 'GB'}, 
+    {name: 'United States', code: 'US'}, 
+    {name: 'United States Minor Outlying Islands', code: 'UM'}, 
+    {name: 'Uruguay', code: 'UY'}, 
+    {name: 'Uzbekistan', code: 'UZ'}, 
+    {name: 'Vanuatu', code: 'VU'}, 
+    {name: 'Venezuela', code: 'VE'}, 
+    {name: 'Viet Nam', code: 'VN'}, 
+    {name: 'Virgin Islands, British', code: 'VG'}, 
+    {name: 'Virgin Islands, U.S.', code: 'VI'}, 
+    {name: 'Wallis and Futuna', code: 'WF'}, 
+    {name: 'Western Sahara', code: 'EH'}, 
+    {name: 'Yemen', code: 'YE'}, 
+    {name: 'Zambia', code: 'ZM'}, 
+    {name: 'Zimbabwe', code: 'ZW'} 
+  ];
+  function list_countries(){
+    const listcountries=[];
+    for (var key in countries) {
+         let obj =JSON.parse(key);
+        listcountries.push(countries[obj].name);
+  }
+  return listcountries;
+}
+
+function list_countries_code(){
+  const listcountries=[];
+  for (var key in countries) {
+       let obj =JSON.parse(key);
+      listcountries.push(countries[obj].code);
+}
+return listcountries;
+}
+ const countriess= list_countries();

```

Comment:
in variables names, you have to choose a meaningful name, nobody will understand what is `countriess` without reading the code